operator/pom.xml

```diff
@@ -22,8 +22,9 @@
         -->
         <resteasy.version>4.7.5.Final</resteasy.version>
         <wildfly.common.version>1.5.4.Final-format-001</wildfly.common.version>
-        <jackson.version>2.13.1</jackson.version>
-        <kubernetes-client.version>5.12.1</kubernetes-client.version>
+        <jackson.version>2.13.2</jackson.version>
+        <jackson.databind.version>2.13.2.2</jackson.databind.version>
```

Comment:
@andreaTP You can safely delete the line above from my understanding, considering that we don't make the use of this property for the Keycloak Operator dependencies. 

Also, please correct me if I'm wrong, but from my understanding `jackson-databind` is a transitive dependency from `kubernetes-client`. I reached to that conclusion after looking into the maven dependency tree:

```
$ cd keycloak/operator &&  mvn dependency:tree | grep -i jackson-databind
[INFO] org.keycloak:keycloak-operator:jar:999-SNAPSHOT
[INFO] +- io.fabric8:crd-generator-api:jar:5.12.1:compile
[INFO] |  +- io.fabric8:kubernetes-client:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-core:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-rbac:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-admissionregistration:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-apps:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-autoscaling:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-apiextensions:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-batch:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-certificates:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-coordination:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-discovery:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-events:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-extensions:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-flowcontrol:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-networking:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-metrics:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-policy:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-scheduling:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-storageclass:jar:5.12.1:compile
[INFO] |  |  +- io.fabric8:kubernetes-model-node:jar:5.12.1:compile
[INFO] |  |  +- com.squareup.okhttp3:okhttp:jar:3.14.9:compile
[INFO] |  |  |  \- com.squareup.okio:okio:jar:1.17.2:compile
[INFO] |  |  +- com.squareup.okhttp3:logging-interceptor:jar:3.14.9:compile
[INFO] |  |  +- org.slf4j:slf4j-api:jar:1.7.30:compile
[INFO] |  |  +- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.13.1:compile
[INFO] |  |  +- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:jar:2.13.1:compile
>>>> [INFO] |  |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.13.1:compile<<<<
[INFO] |  |  +- com.fasterxml.jackson.core:jackson-core:jar:2.13.1:compile
```